templates/web/pages/session/signed_in.tt.html

```diff
@@ -34,6 +34,15 @@ <h3>[% lang("resources") %]</h3>
                 [% lang('faq_for_producers') %]
             </a>
         </li>
+        <li>
+            <a href="[% IF lc == "fr" -%]
```

Comment:
Do I need to remove the dashes here too @stephanegigandet ?

Reply:
yes